### PR TITLE
Add the ability to install pdns_recursor from source.

### PR DESCRIPTION
**Pull request by skemper**
Found that I needed this, since the package on Ubuntu 14.04 is 3.5.x.  Thought it might be useful.  I'm not sure what to do about copyright statements, since my employer isn't already on them, but I think everything else is OK.

### CODE REVIEW

**Comment by skemper** [comment]
There are two issues I'm not sure how to resolve.

1. This introduces a breaking change, with the move of `node['pdns']['authoritative']['source']['path']` to `node['pdns']['source']['path']`.
2. I was unable to run `kitchen test` on this because of problems with the build-essential cookbook, as documented here: https://github.com/opscode-cookbooks/build-essential/issues/41

File: README.md [hunk 1]
```diff
@@ -139,6 +172,7 @@ License & Authors
 - Author:: Aaron Kalin (<aaron.kalin@dnsimple.com>)
 - Author:: Jacobo García (<jacobo.garcia@dnsimple.com>)
 - Author:: Anthony Eden (<anthony.eden@dnsimple.com>)
+- Author:: Stephan Kemper (<stephan.kemper@viasat.com>)
```
Comment (martinisoft) [inline README.md]: Can you remove this line? Happy to list you as a contributor for now, if you would like to help with maintenance we can talk about that and get you on there.

**Comment by martinisoft** [comment]
Thank you for the note on the compile-time issue. I wonder if there is another workaround we could do to fix this problem so testing can resume.

**Comment by martinisoft** [comment]
Ideally, I'd like to get Travis testing this cookbook for us so fixing the compile-time issue would be a major bonus.

**Comment by skemper** [comment]
I have updated the PR with a few things:
  * rebased on top of the new cookbook version
  * removed version numbering updates, so a committer can handle that
  * fixed the bad author line
  * made `kitchen test` work again

**Comment by martinisoft** [comment]
@skemper Awesome! Thank you for sending those changes (and my apologies for the late reply). We will give them another look and see about merging shortly.

**Pull request closed without merging**